packages/fxa-admin-panel/src/components/AccountSearch/Account/index.tsx

```diff
@@ -66,12 +71,29 @@ export const ClearButton = ({
   );
 };
 
-export const DangerZone = () => {
+// gql mutation to update emails table and unverify user's email
+export const UNVERIFY_EMAIL = gql`
+  mutation unverify($email: String!) {
+    unverifyEmail(email: $email)
+  }
+`;
+
+export const DangerZone = ({ email, onCleared }: DangerZoneProps) => {
   const alertWindow = () => {
     window.alert('Implementation coming soon.');
     return;
   };
 
+  const [unverify] = useMutation(UNVERIFY_EMAIL);
+
+  const handleUnverify = () => {
+    if (!window.confirm('Are you sure? This cannot be undone.')) {
+      return;
+    }
+    unverify({ variables: { email: email.email } });
+    onCleared(); // refresh the page
```

Comment:
let's not refresh the page here. we should wait for the response on the mutation. here's an example of using the `onCompleted`/`onError` of a mutation https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/components/UnitRowTwoStepAuth/index.tsx#L54